docs/running-a-node/docker-management.md

```diff
@@ -120,6 +120,10 @@ docker pull nanocurrency/nano:V22.0
 ```
 
 Lastly, we [start up the docker container again](#starting) using the same command but the with new version tag.
+Alteratively, you can use `docker-compose`
+```bash
+docker-compose up
```

Comment:
I have to admit, I haven't tested if it fully works. However, this should work.

Reply:
That might fit better in the Docker Compose section below. One step is missing: You need to edit the tag in the compose file.

Reply:
@DarrenAlex Perhaps when suggesting the alternative, you can just link down to the `#docker-compose` section below, which has examples of files. The addition of a command and note for changing tags between releases would also make sense.